venues/auai.org/UAI/2017/python/config.py

```diff
@@ -8,6 +8,8 @@
 PC = "auai.org/UAI/2017/Program_Committee"
 CONFERENCE = "auai.org/UAI/2017"
 SUBMISSION = "auai.org/UAI/2017/-/blind-submission"
+METADATA = "auai.org/UAI/2017/-/Paper_Metadata"
+ASSIGNMENT = 'auai.org/UAI/2017/-/Matching_Assignments'
```

Comment:
I'm not attached to the name Matching_Assignments, but this invitation is capable of handling assignments from both reviewers and areachairs. Each note has all the information it needs (including the ID of the group that it matches), so having separate invitations would be redundant.

That being said, exactly how to organize it would be up to the conference administrator, and they could choose to do something like this